src/index.ts

```diff
@@ -137,6 +139,33 @@ function processExtensions(definition: Definition): Definition {
             }
         }
 
+        const actionExposes: Expose[] = [];
```

Comment:
```suggestion
        const actionExposes = exposes.filter((e) => e.name === 'action'));
        exposes = exposes.filter((e) => e.name !== 'action'));
```

Reply:
Thanks! I'll change that now

Reply:
I updated the code a bit more, please check again.